v4l2.go

```diff
@@ -311,6 +334,54 @@ func getFrameSize(fd uintptr, index uint32, code uint32) (frameSize FrameSize, e
 	return
 }
 
+func getFrameInterval(fd uintptr, index uint32, code uint32, width uint32, height uint32) (FrameRate, error) {
+	frmivalenum := &v4l2_frmivalenum{}
+	frmivalenum.index = index
+	frmivalenum.pixel_format = code
+	frmivalenum.width = width
+	frmivalenum.height = height
+
+	framerate := FrameRate{}
```

Comment:
```suggestion
	var frameRate FrameRate
```

Reply:
Switched to camelcase and var initialize.